R/PlainTextParam.R

```diff
@@ -0,0 +1,215 @@
+#' @title Store `MsExperiment` and `XcmsExperiment` object as a folder of plain
```

Comment:
Maybe alternative title:

```
@title Store contents of `MsExperiment` and `XcmsExperiment` objects as plain text files
```

I don't think we need to specify in the title that these files will be placed to a specified folder.